MAUI/Button/Customization.md

```diff
@@ -0,0 +1,671 @@
+
+### CornerRadius
+
+The `CornerRadius` property is used to customize the rounded edges in `SfButton` as demonstrated in the following code sample.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" CornerRadius="20">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.CornerRadius = 20;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with cornerradius](images/Button_cornerradius.png)
+
+## Image Customization
+
+The Image can be customized by its ShowIcon, ImageSource, ImageSize and ImageAlignment.
+
+### ShowIcon
+
+You can enable the Icon image using the `ShowIcon` property to know whether any image appears to the `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" ImageSource="Heart.png" ShowIcon="True">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.ImageSource = "Heart.png";
+button.ShowIcon = True;
+
+{% endhighlight %}
+{% endtabs %}
+
+### ImageSource
+
+The `ImageSource` property is used to customize the icon image of `SfButton` by adding a custom image.
+
+N> Enable the `ShowIcon` property to enable the `ImageSource` property. 
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" ImageSource="Heart.png" ShowIcon="True">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.ImageSource = "Heart.png";
+button.ShowIcon = True;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with image with content](images/Button_icon.png)
+
+### ImageSize
+
+The `ImageSize` property is used to customize the width of icon image in `SfButton`.
+
+N> Enable the `ShowIcon` property to enable the `ImageSource` property. 
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" ImageSource="Heart.png" ShowIcon="True" ImageSize="50">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.ImageSource = "Heart.png";
+button.ShowIcon = true;
+button.ImageSize= 50;
+
+{% endhighlight %}
+{% endtabs %}
+
+### ImageAlignment 
+
+The `ImageAlignment` property is used to customize the alignment of icon image in `SfButton`. The following options are available in `ImageAlignment`:
+
+* `Start` - Places the image at the left most of `SfButton`.
+* `End` - Places the image at the right most of `SfButton`.
+* `Top` - Places the image at the top of the text.
+* `Bottom` - Places the image at the bottom of the text.
+* `Left` - Although the flow direction has been applied, it always places the image in the left part of `SfButton`. For example, in the direction of the RTL flow, the image setting will move to the right. Use `Left` alignment to show this in the same left position.
```

Comment:
* `Left` - Although the flow direction has been applied, it always places the image in the left part of `SfButton`. For example, in the direction of the RTL flow, the image setting will move to the right. Use the `Left` alignment to show this in the same left position.